plugins/modules/foreman_ptable.py

```diff
@@ -224,33 +222,17 @@
 )
 
 
-# This is the only true source for names (and conversions thereof)
-name_map = {
-    'template': 'layout',  # the parse_template_* methods stores the "layout" in "template"
-    'layout': 'layout',
-    'locations': 'location_ids',
-    'locked': 'locked',
-    'name': 'name',
-    'organizations': 'organization_ids',
-    'oses': 'os_family',  # the foreman community templates are using oses instead of os_family (which is wrong?)
-    'os_family': 'os_family',
-}
-# Missing parameters:
```

Comment:
are those not missing anymore?

Reply:
They have been missing for a loooong time. I don't think we need to document that.
If anyone needs, PR's are always welcome. ;)